include/k4a/k4a.h

```diff
@@ -1619,8 +1836,8 @@ K4A_EXPORT k4a_buffer_result_t k4a_device_get_raw_calibration(k4a_device_t devic
  * \endxmlonly
  */
 K4A_EXPORT k4a_result_t k4a_device_get_calibration(k4a_device_t device_handle,
```

Comment:
Is it possible to keep original k4a_device_get_calibration? So, the apps already use original k4a_device_get_calibration can still work. This breaking change might be by-design. Just asking.